### ISSUE TITLE
Not working with BP 2.2

### ISSUE THREAD

**Issue by rhisss** [sbrajesh/bp-friends-suggest#2]
Always "We don't have enough details to suggest a friend yet"
Is there a tricky or pseudo way to just show 
- members who are not my friends
- load 5 such members
- as you go clicking Add Friends, show more 5 members

**Comment by nilarocks** [comment]
yes plugin not working. always give a "We don't have enough details to suggest a friend yet" notice
